webapp/src/services/api.ts

```diff
@@ -195,25 +195,21 @@ export const api = createApi({
     updateDataActions: build.mutation<
       UtterancePatch[],
       {
-        persistentIds: UtterancePatch["persistentId"][];
-        newValue: DataAction;
-      } & GetUtterancesQueryState
+        utterancePatch: {
+          persistentId: UtterancePatch["persistentId"];
+          dataAction: DataAction;
+        }[];
+        utteranceQuery: GetUtterancesQueryState;
+      }
```

Comment:
I think it was perfect with ` & GetUtterancesQueryState`, instead of nesting it, so it's uniform with the other endpoints.
```suggestion
      } & GetUtterancesQueryState
```